Firestore/core/src/firebase/firestore/model/transform_operation.cc

```diff
@@ -346,6 +346,12 @@ double AsDouble(const FieldValue& value) {
 
 }  // namespace
 
+NumericIncrementTransform::NumericIncrementTransform(const TransformOperation& op)
+    : TransformOperation(op) {
+  HARD_ASSERT(op.type() == Type::Increment,
+              "Expected increment type; got %s", op.type());
+}
```

Comment:
There's another NumericIncrement constructor on line 309.  Should these be next to each other?

Reply:
Moved.